pandas/core/generic.py

```diff
@@ -4707,6 +4709,45 @@ def get_ftype_counts(self):
         object:dense     1
         dtype: int64
         """
+        warnings.warn("get_ftype_counts is deprecated and will "
```

Comment:
no need to create another function here, this is purely user facing (IOW don't create a _get_ftype_counts). keep the doc-string though.